### PR TITLE
Styles and view template for sponsors, sponsors on front, add payment…

### PR DESCRIPTION
**Pull request by kevinblanco**
- Implemented correct styles for front view on sponsors
- Styles and view php template for sponsors main view. 
- Styles for add payment
- Styles for add session.

### CODE REVIEW

File: sites/all/themes/dccr2015/templates/views/views-view-unformatted--sponsors.tpl.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ * @ingroup views_templates
+ */
+?>
+<?php if (!empty($title)): ?>
+  <h3 class="<?php print(strtolower($title)); ?>"><?php print $title; ?></h3>
+<?php endif; ?>
+
+<div class="sponsors-wrapper">
+	<?php foreach ($rows as $id => $row): ?>
+	  <div<?php if ($classes_array[$id]) { print ' class="' . $classes_array[$id] .'"';  } ?>>
```
Comment (kporras07) [inline sites/all/themes/dccr2015/templates/views/views-view-unformatted--sponsors.tpl.php:16]: @kevinblanco is $classes_array an string?

Comment (kevinblanco) [inline sites/all/themes/dccr2015/templates/views/views-view-unformatted--sponsors.tpl.php:16]: This is a template based on the views unformatted template, it is provided by the views UI "Template information" area. The only change I made from the template was to add the group class to the `<H3>` element to have the ability to style the group headline (place the sponsorship level icon according to the designs). All the other stuff is the same as the template provided by the Views Module.

Comment (kporras07) [inline sites/all/themes/dccr2015/templates/views/views-view-unformatted--sponsors.tpl.php:16]: That's ok for me. I hadn't checked the base template. I just was wondering about the variable name; it sounds like an array.

Comment (kevinblanco) [inline sites/all/themes/dccr2015/templates/views/views-view-unformatted--sponsors.tpl.php:16]: Perfect. It looks like its the row classes

**Pull request merged**